tests/queries/0_stateless/03036_schema_inference_cache_s3_archives.sql

```diff
@@ -1,5 +1,6 @@
 -- Tags: no-fasttest
 -- Tag no-fasttest: Depends on AWS
+SYSTEM DROP SCHEMA CACHE FOR S3;
```

Comment:
It can affect other tests which are being executed in parallel and also will not help to fix this test if some other test fills schema cache in parallel, so if you need this - then better to use `no-parallel` tag. But `no-parallel` should not be used if it is avoidable, as having non-parallel tests slows down tests run. May be instead you could just add `WHERE source ILIKE ...`